### PR TITLE
Debug mode imlplementation

### LINKED ISSUE #99: Support for rp_mode
As per the documentation in http://reportportal.io/docs/How-to-avoid providing rp.mode=debug will avoid local execution reported in project space. However, when I use this option it doesn't work and the execution is reported in project space instead of 'Debug' section of project space.

### PR DESCRIPTION
**Pull request by HardNorth**
Closes #99

### CODE REVIEW

File: examples/test_simple.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+"""Simple example test."""
```
[the remark this answers is not in the document]
Comment (HardNorth) [reply examples/test_simple.py:1]: It's a `pytest` test example, just a simplest test possible. I run it through the whole `pytest` library from an integration test to verify some common things. In this particular case `rp_mode` variable bypassing.

Such approach is better that unit-testing by two reasons:
1. Less code dependencies. You can rewrite the whole agent at once and these tests will still work. I did such a trick recently when I added support of Cucumber 7 for `agent-java-cucumber6` and got only one issue after that: https://github.com/reportportal/agent-java-cucumber6/releases
2. Unit-tests give no guarantees that the system work as a whole. Integration tests give.

File: pytest_reportportal/config.py [hunk 2]
```diff
@@ -36,6 +36,7 @@ def __init__(self, pytest_config):
         self.rp_retries = int(self.find_option('retries'))
         self.rp_uuid = getenv('RP_UUID') or self.find_option('rp_uuid')
         self.rp_verify_ssl = self.find_option('rp_verify_ssl')
+        self.rp_mode = self.find_option('rp_mode')
```
[the remark this answers is not in the document]
Comment (HardNorth) [reply pytest_reportportal/config.py]: Sure

File: examples/test_simple.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (iivanou) [reply examples/test_simple.py:1]: I am completely behind you on integration testing. I do not understand the purpose of this file/test.

File: pytest_reportportal/service.py [hunk 3]
```diff
@@ -137,11 +135,12 @@ def init_service(self,
         self._errors = queue.Queue()
         if self.rp is None:
             self.ignore_errors = ignore_errors
-            self.ignored_attributes = ignored_attributes
+            self.ignored_attributes = ignored_attributes \
```
[the remark this answers is not in the document]
Comment (HardNorth) [reply pytest_reportportal/service.py]: Sure

**Review by iivanou — APPROVED** [review]

**Pull request merged**